### ISSUE TITLE
MRTK v2 , taptoplace and tagalone

### ISSUE THREAD

**Issue by StarWardo** [Microsoft/MixedRealityToolkit-Unity#3653]
Could we add the taptoplace and tagalong scripts from the old toolkit into the new one?

These are very useful for prototyping.

**Comment by Alexees** [comment]
@StarWardo check out this Issue #3576

**Comment by cre8ivepark** [comment]
@StarWardo Please check out Sovlers
https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/mrtk_release/Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers
With Radial View solver or Orbital solver, you can easily achieve tag-along behavior with various configurable options such as lerping timing.

**Comment by StarWardo** [comment]
Thank you for pointing that out, the solvers are indeed what I am looking for. Very usefull!

**Comment by megatran** [comment]
Hello. I checked out the solver examples with different settings but couldn't find one that act similarly as HoloToolkit's TapToPlace. Do you have suggestion on what I should try? Thank you!

**Comment by DrewLovejoy** [comment]
@megatran, I used the "Interactable" script and the "Orbital" solver components.

I then did a little bit of configuring to make it work how I want it to:

I set up the interactable component to be a toggle button by changing the dimensions to equal 2.
I then selected the "InteractableOnToggleReceiver" Event Type.
Then I created a script that has two methods, one that turns on the solver and one that turns off the solver. (This is pretty easy, just get a reference to the "SolverHandler" and then change the boolean "UpdateSolvers" to true or false.)
On the "OnSelect()", I have it set up to call the method that turns on the solver.
On the "On Deselect()", I have it set up to call the method that turns off the solver.
